pkg/interface/src/views/apps/chat/ChatResource.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useRef, useCallback } from "react";
 import { RouteComponentProps } from "react-router-dom";
 import { Col } from "@tlon/indigo-react";
+import _ from 'lodash';
```

Comment:
???

Reply:
Oh this import was missing

Reply:
So `clearUnsent` is going to fail on livenet atm?

Reply:
It doesn't, for reasons I do not understand

Reply:
lodash is so smart

Reply:
Well, `_` is available globally for whatever reason

Reply:
oh lmao this is probably the lodash import rewriter breaking things, but we really shouldn't be using UMD globals ever